lib/inets/src/http_server/httpd.erl

```diff
@@ -507,105 +502,104 @@ _ESI Properties - Requires mod_esi_
 
-- **[](){: #prop_dlog_format } \{disk_log_format, internal | external\}**  
+- [](){: #prop_dlog_format } **`{disk_log_format, internal | external}`**  
   Defines the file format of the log files. See `disk_log` for details. If the
   internal file format is used, the log file is repaired after a crash. When a
   log file is repaired, data can disappear. When the external file format is
   used, `httpd` does not start if the log file is broken. Default is `external`.
 
-- **[](){: #prop_edlog } \{error_disk_log, path()\}**  
+- [](){: #prop_edlog } **`{error_disk_log, path()}`**  
   Defines the filename of the (`m:disk_log`) error log file to be used to log
   server errors. If the filename does not begin with a slash (/), it is assumed
   to be relative to the `server_root`.
 
-- **[](){: #prop_edlog_size } \{error_disk_log_size, \{MaxBytes,
-  MaxFiles\}\}**  
+- [](){: #prop_edlog_size } **`{error_disk_log_size, {MaxBytes, MaxFiles}}`**  
   `MaxBytes = integer()` and `MaxFiles = integer()`. Defines the properties of
   the (`m:disk_log`) error log file. This file is of type wrap log and max bytes
   is written to each file and max files is used before the first file is
   truncated and reused.
 
-- **[](){: #prop_sdlog } \{security_disk_log, path()\}**  
+- [](){: #prop_sdlog } **`{security_disk_log, path()}`**  
   Defines the filename of the (`m:disk_log`) access log file logging incoming
   security events, that is, authenticated requests. If the filename does not
   begin with a slash (/), it is assumed to be relative to the `server_root`.
 
-- **[](){: #prop_sdlog_size } \{security_disk_log_size, \{MaxBytes,
-  MaxFiles\}\}**  
+- [](){: #prop_sdlog_size } **`{security_disk_log_size, {MaxBytes, MaxFiles}}`**  
   `MaxBytes = integer()` and `MaxFiles = integer()`. Defines the properties of
   the `m:disk_log` access log file. This file is of type wrap log and max bytes
   is written to each file and max files is used before the first file is
   truncated and reused.
 
-- **[](){: #prop_tdlog } \{transfer_disk_log, path()\}**  
+- [](){: #prop_tdlog } **`{transfer_disk_log, path()}`**  
   Defines the filename of the (`m:disk_log`) access log file logging incoming
   requests. If the filename does not begin with a slash (/), it is assumed to be
   relative to the `server_root`.
 
-- **[](){: #prop_tdlog_size } \{transfer_disk_log_size, \{MaxBytes,
-  MaxFiles\}\}**  
+- [](){: #prop_tdlog_size } **`{transfer_disk_log_size, {MaxBytes, MaxFiles}}`**  
   `MaxBytes = integer()` and `MaxFiles = integer()`. Defines the properties of
   the `m:disk_log` access log file. This file is of type wrap log and max bytes
   is written to each file and max files is used before the first file is
   truncated and reused.
 
 [](){: #props_auth }
 
-_Authentication Properties - Requires mod_auth_
+### Authentication Properties - Requires mod_auth
 
 [](){: #prop_dri }
 
-_\{directory, \{path(), \[\{property(), term()\}]\}\}_
+```erl
```

Comment:
Thank you for your help with documentation!
I have just one comment though, please replace all of ```erl with ```erlang, as ```erl is not used across otp repository

Best regards, Konrad